client/hosting/overview/components/migration-overview/style.scss

```diff
@@ -0,0 +1,47 @@
+.migration-overview__container {
+	& > .migration-overview__content {
+		margin: 0 auto;
+	}
+	.migration-overview__header {
+		margin-bottom: 40px;
+	}
+	.migration-started-difm {
+		display: flex;
+		flex-direction: column;
+		gap: 1rem;
+		margin: 0 auto;
+		padding-bottom: 2rem;
+		max-width: 720px;
+		.migration-started-difm__title {
+			font-size: 1rem;
+			font-weight: 600;
+		}
+
+		.migration-started-difm__list {
+			display: flex;
+			flex-direction: column;
+			gap: .75rem;
+			margin: 0;
+		}
+
+		.migration-started-difm__item {
+			display: flex;
+			gap: .75rem;
+			font-size: 1rem;
+			align-items: center;
+		}
+		.migration-started-difm__icon {
+				fill: #3858E9;
```

Comment:
formatting is weird here

Reply:
Fixed [717f480](https://github.com/Automattic/wp-calypso/pull/95963/commits/717f480ecd2326edf7626ff0b0999d3a56d0f8bd)